### PR TITLE
Increased alcohol/nicotine tolerance

### PR DESCRIPTION
**Pull request by lessthnthree**
## About The Pull Request

Increases the alcohol tolerance for mobs, modifies the effects that happen at higher levels. You can still get unintelligibly drunk but it will take downing hard alcohol. High consumption of wine or mixed drinks will get you tipsy, but not full strength slurring unless you're drinking constantly. Similar changes for cigarettes (which aside from low tolerance was applying the effect twice for some reason.)

## How This Contributes To The Skyrat Roleplay Experience

Bar roleplay is hindered by how quickly characters get drunk with the /tg/ thresholds for alcohol. 

For an RP server, it's just annoying. When others who use smokes stop for an RP scene you end up clogging up the chat log with "So-n-So coughs" messages. Additionally, the jitteriness fucks with layering and such, making some scenes wonk out. 

The mood debuff (which affects your speed) and the other side effects are pretty much mechanically punishing enough and don't really damage the RP side of things.

I've known plenty of smokers in my life. I have yet to see one not smoke for a minute and start having convulsions and coughing up a storm.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

Wine will cap out at ~24. (Drunk starts at 21 now instead of 6.)

Vodka caps out around 54.

image

</details>

## Changelog

:cl: LT3
qol: Crew have learned how to properly hold their alcohol
qol: Crew can smoke a cigarette without falling into an instant coughing fit
fix: Fixed nicotine addiction side effects being applied twice each time
/:cl:

### CODE REVIEW

File: code/__DEFINES/~skyrat_defines/mobs.dm [hunk 1]
```diff
@@ -5,6 +5,9 @@
 /// Used for Nanite Slurry vomit. The mob will vomit a nanite puddle.
 #define VOMIT_NANITE 3
 
+/// How much booze a mob can have before getting drunk.
+#define TIPSY_THRESHOLD 21
```
Comment (Iamgoofball) [inline code/__DEFINES/~skyrat_defines/mobs.dm]: Please document on the comment that the vanilla definition is 6

**Review by Gandalf2k15 — DISMISSED** [review]
Can you try to figure out a way to avoid commenting out clode blocks? Not a fan of this method anymore

**Comment by lessthnthree** [comment]
> Can you try to figure out a way to avoid commenting out clode blocks? Not a fan of this method anymore

Yep, done.

**Comment by Nerev4r** [comment]
Real shit, can we tack hunger onto this? It would be a lot better for chef mains if the hunger rate was halved.

**Comment by glitchbow** [comment]
God fucking bless this PR, lord only knows that when I haven written scenes with smoker chars the amount of "[character] coughs! (x20)"'s I've seen is insane, in addition to that one goofy message about thinking you are free of your addiction only to return to it again.

I almost want to suggest a separate PR just for a chainsmoker trait that adds a permanent debuff to your character but almost entirely minimizes coughing completely.

**Review by hugmeorelse — APPROVED** [review]

**Pull request merged**